documentation/restore.md

```diff
@@ -0,0 +1,26 @@
+# Restore Puppet Enterprise using the PEADM module
+
+Once you have a [backup](backup.md) you can restore a PE primary server. It is important to highlight that you can not use the `peadm::restore` plan with a backup that was not created with the `peadm::backup` plan.
+
+As in the `peadm::backup` plan, you can choose what you want to restore by specifying the parameter `restore`
+
+Example:
+
+```
+# restore_params.json
+
+{
+  "restore": {
+    "orchestrator": false,
+    "puppetdb": false,
+    "rbac": true,
+    "activity": false,
+    "ca": false,
+    "classifier": false,
+  },
+  "input_file": "/tmp/path"
+}
+```
+To run the `peadm::restore` plan with our custom parameters file, we can do:
+
+    bolt plan run peadm::restore --params @restore_params.json
```

Comment:
What will happen when `peadm::restore` executes?
Is there any downtime?

Reply:
It will stop servivces so there is downtime for duration of restore it will reinitialise replicas too so we want to note that.

We probably want to note a temporary escalation is put in to allow Primary to talk to PuppetDB on xl setup

Reply:
Not sure "temporary escalation" is clear for everyone. Do you mean a temporary permission change? For what user?
Also, we should describe the fact that downtime is to be expected and what services will go down for how long, for both backup and restore.

Reply:
I made some adjustments based on this feedback, it starts on Line 5. 

But specifically, I added this new paragraph

https://github.com/puppetlabs/puppetlabs-peadm/pull/294/files#diff-e9069702983897147b48e019b1eb79a368506db0d32e09a65c7cfac83f511df6R17

Reply:
@mcka1n Can we add the note that this can only be exceuted on the primary?